flink-kubernetes/src/test/java/org/apache/flink/kubernetes/KubernetesPodTemplateTestUtils.java

```diff
@@ -60,7 +60,7 @@ public static File getPodTemplateFile() {
     public static Container createInitContainer() {
         return new ContainerBuilder()
                 .withName("artifacts-fetcher")
-                .withImage("testing-init-image")
+                .withImage("busybox:latest")
```

Comment:
Maybe we do not have to change this image since it is used in a test. And we do not expect it should exist.

Reply:
> Maybe we do not have to change this image since it is used in a test. And we do not expect it should exist.

fix it